src/jdk.compiler/share/classes/com/sun/tools/javac/jvm/Target.java

```diff
@@ -226,4 +226,10 @@ public boolean obsoleteAccStrict() {
     public boolean optimizeOuterThis() {
         return compareTo(JDK1_18) >= 0;
     }
+
+    /** Releases prior to JDK 23 expect a less precise SwitchBootstraps.typeSwitch signature on the selectorType
+     */
+    public boolean releaseAfterJEP455() {
```

Comment:
Perhaps consider a more descriptive method name instead of referring to the JEP number, for consistency with the other method names in this class.